_dev/src/assets/json/translations/en/ui.json

```
@@ -545,7 +545,10 @@
 
   "smartShoppingCampaignList": {
     "tableTitle": "Campaign performance",
-    "xCampaign": "No campaigns | {0} campaign | {0} campaigns"
+    "xCampaign": "No campaigns | {0} campaign | {0} campaigns",
+    "performanceMax" : "Performance Max",
+    "smartShoppingCampaign": "Smart Shopping Campaign",
+    "campaignType": "Type : {0}"
```

In English, there's no space before the colon

```suggestion
    "campaignType": "Type: {0}"
```